classes/publication/Collector.php

```diff
@@ -135,7 +143,22 @@ public function getQueryBuilder(): Builder
             $qb->offset($this->offset);
         }
 
-        $qb->orderBy('p.publication_id', 'asc');
+        if ($this->orderByVersion) {
+            $orderCase = "CASE p.version_stage ";
+            foreach (VersionStage::cases() as $case) {
+                $orderCase .= "WHEN " . DB::getPdo()->quote($case->value) . " THEN " . $case->order() . " ";
```

Comment:
@defstat, did you forget to add the change to `VersionStage` to add the `order` function used here?